### PR TITLE
able to print bollinger bands with SPY

### PR DESCRIPTION
**Pull request by kraigstrong**
Paired. You all saw what it did

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "python.pythonPath": "${workspaceFolder}/.venv/bin/python"
+  "python.pythonPath": "C:\\go-broke\\.venv\\Scripts\\python.exe",
```
Comment (kraigstrong) [inline .vscode/settings.json:2]: This is obviously not platform independent. Should we .gitignore this file?

Comment (cdaringe) [reply .vscode/settings.json:2]: ya, good call

**Comment by cdaringe** [comment]
feel free to merge and we can just fix that file later

File: go_broke/main.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import go_broke.aquisition as aq
+import data_formatter
```
Comment (cdaringe) [inline go_broke/main.py:2]: does this work?  do you need to do `.data_formatter` or `go_broke.data_formatter`?  i figured doing this wont find that file in the PYTHONPATH

Comment (kraigstrong) [reply go_broke/main.py:2]: The IDE complained that it couldn't find data_formatter, but it did run just fine. It did the same thing with importing bollinger.

Comment (absheri) [reply go_broke/main.py:2]: If you use from data_formatter import * you can directly use the functions defined in the file.

**Pull request merged**